src/ucs/debug/memtrack.c

```diff
@@ -234,6 +234,16 @@ char *ucs_strndup(const char *src, size_t n, const char *name)
     return str;
 }
 
+int ucs_vasprintf(const char *name, char **strp, const char *fmt, va_list ap)
```

Comment:
we avoid using asprintf because it's not portable

Reply:
ok, will remove it.

Reply:
done
sorry, need to do force-push to remove commit